src/package.xml

```diff
@@ -1,20 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Package xmlns="http://soap.sforce.com/2006/04/metadata">
     <types>
-        <members>CountryStateDistrict</members>
+        <members>AccountTriggerHandler</members>
         <name>ApexClass</name>
     </types>
     <types>
-        <members>Country__c</members>
-        <members>District__c</members>
-        <members>State__c</members>
-        <name>CustomObject</name>
-    </types>
-    <types>
-        <members>Country__c</members>
-        <members>District__c</members>
-        <members>State__c</members>
-        <name>CustomTab</name>
+        <members>Trigger_Operations</members>
+        <name>ApexTrigger</name>
     </types>
```

Comment:
Didn't include the new record types on account and the new values for the "type" picklist field on the account.